maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -310,6 +353,44 @@ private void downloadOWAs(File targetDirectory, DistroProperties distroPropertie
 		}
 	}
 
+	private void setConfigFolder(Server server, DistroProperties distroProperties) throws MojoExecutionException {
+		if (distroProperties != null) {
+			File configDir = new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_CONFIGURATION);
+			configDir.mkdir();
+			downloadConfigs(distroProperties, configDir);
+			File referenceApplicationFile = new File(configDir, "referenceapplication-distro.owa");
+			if(!referenceApplicationFile.exists()) {
+				return;
+			}
+			try {
+				ZipFile zipFile = new ZipFile(referenceApplicationFile);
+				zipFile.extractAll(configDir.getPath());;
+				for(File file: Objects.requireNonNull(configDir.listFiles())) {
+					if(file.getName().equals("openmrs_config")) {
+						FileUtils.copyDirectory(file, configDir);
+					}
+					FileUtils.deleteQuietly(file);
+				}
+				referenceApplicationFile.delete();
+			}
+			catch (ZipException | IOException e) {
+				throw new RuntimeException(e);
+			}
+		}
+	}
+
+	private void downloadConfigs(DistroProperties distroProperties, File configDir)
+			throws MojoExecutionException {
+		List<Artifact> configs = distroProperties.getConfigArtifacts();
+		wizard.showMessage("Downloading Configs...\n");
+		if (!configs.isEmpty()) {
+			for (Artifact config : configs) {
+				wizard.showMessage("Downloading Config: " + config);
+				owaHelper.downloadOwa(configDir, config, moduleInstaller);
```

Comment:
Are `owa` s part of O3? I am looking at owaHelper

Reply:
I used those to download the maven package. Is there a another way to do it??

Reply:
Which maven package?

Reply:
Did you see the above question?